src/main/scala/extractionStreams/WikiExtractionStream.scala

```diff
@@ -0,0 +1,14 @@
+package extractionStreams
+
+import akka.stream.Materializer
+import akka.stream.scaladsl.RunnableGraph
+
+/**
+ * Created by nico on 13/07/15.
+ */
+abstract class WikiExtractionStream[T] {
+  val stream: RunnableGraph[T]
+  def run()(implicit materializer: Materializer): Unit = {
```

Comment:
1. i can call run with `run()(materializer)` or like this `run()` if i have a `implicit val materializer` in my scope.

2.yes sure